src/main/kotlin/com/teamdev/jxbrowser/gradle/JxBrowserExtension.kt

```diff
@@ -112,6 +112,14 @@ public open class JxBrowserExtension(private val project: Project) {
      */
     public val win32: Provider<String> = artifact("win32")
 
+    /**
+     * Returns a dependency notation for the `jxbrowser-win64-arm`,
+     * an artifact with Chromium Windows ARM 64-bit binaries.
+     *
+     * As for now, Windows ARM is only supported in JxBrowser 8.x.x.
+     */
+    public val win64Arm: Provider<String> = artifact("win64-arm")
```

Comment:
How about just `winArm`?

Reply:
For the sake of saving consistency with `win64`/`win32` let's use this naming.